sources/kube.go

```diff
@@ -147,11 +147,11 @@ func (self *kubeSource) getPodInfo(nodeList *nodes.NodeList) ([]api.Pod, error)
 	var (
 		wg sync.WaitGroup
 	)
-	for _, pod := range pods {
+	for index := range pods {
 		wg.Add(1)
 		go func(pod *api.Pod) {
 			defer wg.Done()
-			for _, container := range pod.Containers {
+			for index, container := range pod.Containers {
 				rawContainer, err := self.getStatsFromKubelet(pod, container.Name)
 				if err != nil {
 					// Containers could be in the process of being setup or restarting while the pod is alive.
```

Comment:
Is the log line below right? It will display "<container name>/<namespace> <pod>"